### ISSUE TITLE
fails to run under python 3.8 

### ISSUE THREAD

**Issue by fischercer** [CastXML/pygccxml#113]
When I try to use pygccxml under python 3.8 on Manjaro I get the following error:

```txt
...
    declarations = parser.read_files(files, compilation_mode)
  File "/usr/lib/python3.8/site-packages/pygccxml/parser/project_reader.py", line 264, in read_files
    return self.__parse_file_by_file(files)
  File "/usr/lib/python3.8/site-packages/pygccxml/parser/project_reader.py", line 319, in __parse_file_by_file
    start_time = time.clock()
AttributeError: module 'time' has no attribute 'clock'
```
The clock method was removed in Python 3.8 as it was deprecated since python 3.3. Therefore, this issues arises.

**Comment by drekeuk** [comment]
We are also hitting this issue using Python 3.8. It looks like this issue is resolved in the develop branch when do you expect to merge into master and make a new release?

**Comment by tomhenderson** [comment]
I can confirm that develop branch currently works for the ns-3 project and Python 3.8 (tested on Ubuntu 20.04).

**Comment by iMichka** [comment]
I am working on updating the build pipelines https://github.com/gccxml/pygccxml/pull/112.
Once this is done I will do a final review and try to tag a new release.

**Comment by billhoffman** [comment]
Is there a status on this?

**Comment by sumagnadas** [comment]
When is a new version coming out?

**Comment by iMichka** [comment]
Soon. I am currently reviewing the last issues. This one is fixed on develop so I'll close it.
